model-optimizer/extensions/middle/ONNXResize11ToInterpolateV4.py

```diff
@@ -0,0 +1,166 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import logging as log
+import numpy as np
+
+from extensions.ops.Cast import Cast
+from extensions.ops.elementwise import Add, Div, Mul
+from extensions.ops.interpolate import Interpolate
+from mo.front.common.layout import get_depth_dim, get_height_dim, get_width_dim
+from mo.front.common.partial_infer.utils import int64_array, float_array
+from mo.front.tf.graph_utils import create_op_with_const_inputs
+from mo.middle.replacement import MiddleReplacementPattern
+from mo.graph.graph import Graph, Node, rename_nodes
+from mo.ops.const import Const
+from mo.ops.shape import Shape
+from mo.ops.strided_slice import StridedSlice
+
+
+def convert_mode(onnx_mode: str) -> str:
+    return {'nearest': 'nearest', 'linear': 'linear_onnx', 'cubic': 'cubic'}[onnx_mode]
+
+
+def replace_resize(graph: Graph, resize: Node):
+    log.debug("Converting of ONNX Resize-11 to Interpolate-4 is triggered.")
+
+    input_shape = resize.in_port(0).data.get_shape()
+    input_rank = len(input_shape)
+    if input_rank not in {4, 5}:
+        log.warning('The input shape is not 4D or 5D for op with name {}'.format(resize.name))
```

Comment:
Use `resize_name = resize.soft_get('name', resize.id)` instead of `resize.name` everywhere

Reply:
Fixed.